tests/core/training/test_interactive.py

```diff
@@ -756,7 +757,8 @@ def test_retry_on_error_success(monkeypatch: MonkeyPatch):
             f"The bot wants to run 'action_unlikely_intent' "
             f"to indicate that the last user message was unexpected "
             f"at this point in the conversation. "
-            f"Check out IntentTEDPolicy ({DOCS_URL_POLICIES}/#intent-ted-policy) "
+            f"Check out UnexpecTEDIntentPolicy "
+            f"({DOCS_URL_POLICIES}/#unexpected-intent-policy) "
```

Comment:
the slash before the `#` looks weird, doesn't it? Same above

Reply:
You are correct. It shouldn't have been in the first place. Nothing to do with renaming.